Header header
float32 velocity
float32 linear_acceleration_x
float32 linear_acceleration_y
float32 average_distance
float32 time_to_collision
bool forward_collision_warning
bool autopilot_enabled
bool forced_disengagement
float32 predicted_collision_frequency
float32 safety_score 